modules/end-user-guide/partials/proc_allowing-users-to-define-workspace-creation-strategy.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// creating-a-workspace-from-a-remote-devfile
+
+[id="allowing-users-to-define-workspace-creation-strategy_{context}"]
+= Allowing users to define workspace creation strategy
+
+
+As a developer, you can configure {prod-short} to create a new workspace each time it accepts a factory URL, or to reuse the existing workspace if a user already has one. {prod-short} currently supports the following options: 
```

Comment:
```suggestion
As a developer, you can configure {prod-short} to create a new workspace each time it accepts a factory URL, or to reuse the existing workspace if a user already has one.

{prod-short} currently supports the following options: 
```

Reply:
Minimalism! Sure thing!
Thank you so much, Rolfe!